lib/internal/util.js

```diff
@@ -217,3 +219,62 @@ module.exports = exports = {
   // default isEncoding implementation, just in case userland overrides it.
   kIsEncodingSymbol: Symbol('node.isEncoding')
 };
+
+const kCustomPromisifiedSymbol = Symbol('util.promisify.custom');
+const kCustomPromisifyArgsSymbol = Symbol('customPromisifyArgs');
```

Comment:
What's the motivation for keeping this private? It's obviously useful in a number of places.

Reply:
https://github.com/nodejs/node/pull/12442/files#diff-a43208147d795be6dd3517c53226e37dR280 ?

Reply:
but that's still on internal/util, not accessible to user code. The other one(kCustomPromisifiedSymbol) is available as `util.promisify.custom`.

Reply:
@syrnick See #14007.